src/composables/accounts.ts

```diff
@@ -87,15 +90,26 @@ const accounts = computed((): IAccount[] => {
   }
 
   const idxList = Object.fromEntries(PROTOCOL_LIST.map(((protocol) => [protocol, 0])));
+  let airGapIdx = 0;
```

Comment:
I don't think having separate variables for each account types is reasonable. Maybe we could have `idxList` with 2 levels instead of 1? This way we would access it in a following way: `idxList[account.protocol][account.type]`.